django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.db.models.expressions import Default
+from django.utils.translation import gettext_lazy as _
+
+__all__ = ('BigSerialField', 'SmallSerialField', 'SerialField')
+
+
+class SerialFieldMixin:
+    db_returning = True
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **{
+            'editable': False,
+            **kwargs,
+            'blank': True,
+            'default': Default(),
+            'unique': True,
+        })
```

Comment:
`serial` columns in PostgreSQL have a `NOT NULL` constraint. You may want to force `null` to be `False`